test/test_files/function/gds/shortest_paths_large.test

```diff
@@ -15,27 +15,56 @@
 3|1000
 4|10000
 
+# The first output tuples 11 comes from the fact that we also output 0-length paths, i.e., the from from source node
+# to source node. Both for that path and the 10 paths of length 1, we output an empty pathNodeIDs, since there are no
+# intermediate nodes.
+-CASE SingleSPPathsLargeSingleLabel
+-STATEMENT PROJECT GRAPH PK (person1, knows11)
+           MATCH (a:person1) WHERE a.ID = 1
+           CALL single_sp_paths(PK, a, 4, true)
+           RETURN size(pathNodeIDs), count(*);
+-PARALLELISM 8
```

Comment:
Remove -PARALLELISM from this file. Parallelism is enabled by default. And we should deprecate this configuration in test file.

Reply:
Done, but I wouldn't deprecate this. We might want to test cases with 1 or very large number of threads.